src/main/java/ru/practicum/shareit/item/service/ItemServiceImpl.java

```diff
@@ -0,0 +1,91 @@
+package ru.practicum.shareit.item.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import ru.practicum.shareit.exception.NotFoundException;
+import ru.practicum.shareit.item.dto.ItemDto;
+import ru.practicum.shareit.item.model.Item;
+import ru.practicum.shareit.item.storage.impl.ItemRepository;
+import ru.practicum.shareit.user.model.User;
+import ru.practicum.shareit.user.storage.impl.UserRepository;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static ru.practicum.shareit.item.mapper.ItemMapper.toItem;
+import static ru.practicum.shareit.item.mapper.ItemMapper.toItemDto;
+import static ru.practicum.shareit.item.mapper.ItemMapper.updateItem;
+
+
+@Service
+@RequiredArgsConstructor
+public class ItemServiceImpl implements ItemService {
+    private final ItemRepository itemRepository;
+    private final UserRepository userRepository;
+
+    @Override
+    public List<ItemDto> getAll(Long userId) {
+        return itemRepository.getAll().stream()
+                .filter(x -> userId.equals(x.getOwner().getId()))
+                .map(s -> toItemDto(s))

```

Comment:
Можно упростить с помощью [method reference](https://www.baeldung.com/java-method-references)
```suggestion
                .map(ItemMapper::toItemDto)
```